snapcraft/project/_project.py

```diff
@@ -53,14 +54,28 @@ def __init__(
 
         super().__init__(target_deb_arch, debug, work_dir=work_dir)
 
+        self.local_plugins_dir = self._get_local_plugins_dir()
+
     def _get_snapcraft_assets_dir(self) -> str:
+        if not self.info:
```

Comment:
Can you add a comment on when this situation takes place?

Reply:
Comment added.